run.sh

```diff
@@ -9,7 +9,13 @@ create)
 migrate_and_seed)
     echo "running migrate and seed"
     bundle exec rake db:migrate db:seed
+    RETURN_CODE=$?
+    while [ $RETURN_CODE -eq 1 ]; do
```

Comment:
I could see a couple issues:
1. Probably want to retry while the code is different than `0`, rake return code might not always return 1 in case of failure.
2. Probably only want to retry a limited amount of times

I think this PR is resulting from one of our migration timing out. We fail migration that takes longer than a couple seconds to avoid agressive table lock, typically when modifying a column defaults. 
I don't think this would fixed the issue we had since it would keep retrying the migration.

Reply:
@StupidCodeFactory thank you for input.
I just changed eq to gt than 0 as per suggestion.
We want to retry as long as it fails because we already have new containers on the instances which won't serve any traffic unless you can either perform manual failback or the migration will succeed.